ProjectSettings/TagManager.asset

```diff
@@ -19,32 +19,31 @@ TagManager:
   - Smells
   - Water
   - UI
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
-  -
+  - 
```

Comment:
I don't actually get why because nothing really changed here, but I was getting Unable to parse file ProjectSettings/TagManager.asset: [Parser Failure at line 49: Expect ':' between key and value within mapping]